python/lib/communication/dmod/communication/maas_request.py

```diff
@@ -484,10 +494,12 @@ def to_dict(self) -> dict:
 
 
 class MaaSRequestResponse(Response, ABC):
-
     def __init__(self, success: bool, reason: str, message: str = '', data=None):
         super().__init__(success=success, reason=reason, message=message, data=data)
 
+
+class ModelExecRequestResponse(MaaSRequestResponse, ABC):
```

Comment:
Why does this subclass ABC?

Reply:
In my mind, the `ModelExec`-layer needed to be added to differentiate between general user-initiated requests (i.e., the `MaaS`-layer request/response) and user-initiated requests specifically for running a model.  To be consistent, I thought both the request and the response subtypes should be included.

But even with this new separation, these are both still abstractions.   Only some subtype of `ModelExecRequestResponse` should ever be directly instantiated, or there would be some ambiguity (or at least inconsistency with the other request/response usages).   So I added the explicit `ABC` subclassing to reenforce the intended design.